tests/test_converter.py

```diff
@@ -132,6 +132,29 @@ def patient_resource() -> Patient:
     return Patient(**patient_data)
 
 
+@pytest.fixture
+def patient_dict() -> Dict:
+    return {
+        'hospitalId': 'hospa',
+        'encounterId': 'ENC1111',
+        'patientId': 'MRN1234',
+        'sex': 'male',
+        'dateOfBirth': '1951-07-06',
+        'givenName': 'Thomas',
+        'familyName': 'Jones',
+        'ssn': '123-45-6789',
+        'state': 'MN',
+        'rowNum': 1,
+        'groupByKey': 'MRN1234',
+        'timeZone': 'US/Eastern',
+        'tenantId': 'sample-tenant',
+        'streamType': 'live',
+        'emptyFieldValues': ['empty', '\\n'],
+        'filePath': '/Users/hammadkhan/Projects/LinuxForHealth/CsvToFHIR/tests/resources/csv/2022-02-18-patient-fwf.dat',
```

Comment:
Hardcoded personal path should be removed.

Reply:
This change is completely backwards compatible, in fact the original behavior is still the default.